ci/github-script/commits.js

```diff
@@ -179,9 +179,14 @@ module.exports = async function ({ github, context, core }) {
         }
 
         core.summary.addRaw('<details><summary>Show diff</summary>')
-        core.summary.addRaw('\n\n```diff', true)
-        core.summary.addRaw(truncated.join('\n'), true)
-        core.summary.addRaw('```', true)
+        core.summary.addCodeBlock(
```

Comment:
> Previously, when the diff contained a context line with ```, this would end the code block and entirely break the markdown rendering.

Another option is to use a longer code fence, since the code block only ends when it finds an unindented `` ``` `` fence of the same length as the opening fence.

The only way to do this _safely_ is to extract the longest `` `+ `` substring from the diff and concatenate it with ` ```diff `, but the simpler option is to hard-code an unreasonably long code fence.

E.g.
```js
core.summary.addRaw('\n\n`````````````diff', true)
core.summary.addRaw(truncated.join('\n'), true)
core.summary.addRaw('`````````````', true)
```